app/templates/components/ui-table/cell/events/view/sessions/cell-session-title.hbs

```diff
@@ -1,5 +1,5 @@
 {{this.record}}
-<div class="hidden ui divider"></div>
+<div class="hidden ui divider" style="margin: 0.3em"></div>
```

Comment:
instead of hidden divider use a line break <br>. And see if it holds the requirement.

Reply:
Thank you @maze-runnar for your valuable suggestion. I will try to use <br> tag instead of hidden divider.